### PR TITLE
feat: des-2697 s-app-card & bootstrap 3 has-row--col-stretch-content-y

### PR DESCRIPTION
**Pull request by wesleyboar**
## Overview

Style "app cards" for (initially for DesignSafe client).

> [!IMPORTANT]
> This is a structure only UI pattern. [DesignSafe client will import and skin it.](https://github.com/DesignSafe-CI/portal/pull/1168)

> [!NOTE]
> **To Do**
> - [x] make demo responsive, not flexible

## Related

- [DES-2697](https://tacc-main.atlassian.net/browse/DES-2697)
- https://github.com/DesignSafe-CI/portal/pull/1168

## Changes

- **added** `.s-app-card` a.k.a `.app-card`
- **fixed** `.has-row--col-stretch-content-y` in Bootstrap 3

## Testing

1. Open http://localhost:3000/components/preview/s-app-card.
2. See how un-skinned it looks.
3. But notice it has a structure / layout.
4. Compare to [skinned version](https://github.com/DesignSafe-CI/portal/pull/1168).

### Static

| Raw Screenshot | Highlighted Grid & Flex |
| - | - |
| structure - unhighlighted | structure - highlighted |

### Responsiveness

https://github.com/TACC/Core-Styles/assets/62723358/58042f40-2f8c-4b5f-8937-508c50993712

<details><summary><h3>Client Use Case</h3></summary>

https://github.com/DesignSafe-CI/portal/assets/62723358/fd525337-7c83-4f5e-98c5-9ee5bdb73db4

</details>

### CODE REVIEW

File: src/lib/_imports/components/bootstrap.row.css [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
 }
+
+/* BOOTSTRAP 3 */
+/* Add this class to any containing element */
+/* NOTE: DesignSafe CMS Admin can not add class on Bootstrap 3 grid elements */
+/* https://stackoverflow.com/a/22892773/11817077 */
+.has-row--col-stretch-content-y .row:has([class^="col-"]) {
+  display: flex;
+  flex-wrap: wrap;
+}
```
Comment (wesleyboar) [inline src/lib/_imports/components/bootstrap.row.css:16]: Used on https://www.designsafe-ci.org/testing-pages/image-plugin-email-link/?edit (search the markup for it).

**Review by R-Tomas-Gonzalez — COMMENTED** [review]
I don't have access to designsafe, but I went ahead and looked at the ui on the pattern library. 

My only thing is, the words seem to bounce around quite a bit. Might be good to put a width on the cards? If that's not a concern, then I approve! :)


https://github.com/TACC/Core-Styles/assets/63771558/b38bb32b-847b-42cd-804e-1fe7652a514c

**Comment by wesleyboar** [comment]
@R-Tomas-Gonzalez, reasonable concern. If you cannot access https://www.designsafe-ci.org/testing-pages/image-plugin-email-link/ (client use case) yet, then see this video (newly added to DesignSafe-CI/portal#1168) of it on a test page. Each card is in a Bootstrap grid column, which is responsive (same width until breakpoint) not flexible (like Core-Styles demo). Ooh… the demo could be responsive too, using `o-grid`. Task added.

https://github.com/DesignSafe-CI/portal/assets/62723358/fd525337-7c83-4f5e-98c5-9ee5bdb73db4

File: src/lib/_imports/trumps/s-app-card/s-app-card.hbs
Comment (wesleyboar) [inline src/lib/_imports/trumps/s-app-card/s-app-card.hbs:1]: @R-Tomas-Gonzalez I used Bootstrap for grid. It burns my eyes. I can't look anymore. Does it work?

_**Reminder**: Only designers and devs are eagle-eyeing text while shrinking and growing windows. Typical user interaction is a layout switch upon orientation change or window maximize._

https://github.com/TACC/Core-Styles/assets/62723358/58042f40-2f8c-4b5f-8937-508c50993712

**Review by R-Tomas-Gonzalez — APPROVED** [review]
LGTM

**Pull request merged**